packages/xs/xapi-rrd.1.7.0/opam

```diff
@@ -9,26 +9,28 @@ tags: [
 ]
 build: [
   ["dune" "build" "-p" name "-j" jobs]
-  ["dune" "runtest" "-p" name] {with-test}
+]
+build-test: [
+  ["dune" "runtest" "-p" name]
 ]
 depends: [
   "ocaml"
   "dune" {build & >= "1.4.0"}
   "base-bigarray"
   "base-unix"
-  "ppx_deriving_rpc"
-  "rpclib"
+  "rpc" {>= "1.9.51" & < "5.0.0"} | ("rpclib" & "ppx_deriving_rpc")
```

Comment:
I think we could've dropped `rpc` from here completely, but lets leave it as is now.

Reply:
We did, I need to change the opam file in xcp-rrd